test/spec/modules/liveIntentAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,309 @@
+import liAnalytics from 'modules/liveintentAnalyticsAdapter';
+import { expect } from 'chai';
+import { server } from 'test/mocks/xhr.js';
+import { auctionManager } from 'src/auctionManager.js';
+
+let utils = require('src/utils');
+let instanceId = '77abbc81-c1f1-41cd-8f25-f7149244c800';
+let url = window.location.href;
+let sandbox;
+let clock;
+let now = new Date();
+
+let events = require('src/events');
+let constants = require('src/constants.json');
+let auctionId = '99abbc81-c1f1-41cd-8f25-f7149244c897'
+
+const config = {
+  provider: 'liveintent',
+  options: {
+    bidWonTimeout: 2000,
+    sampling: 1
+  }
+}
+
+let args = {
```

Comment:
we wanted to make sure the event is parsed correctly and compare it to the result using its values

Reply:
Problem is that this will not catch if prebid changes the schema of the event. Though we had a look with Viktor and seems like most adapters to it this way -- so it's fine